src/coreclr/src/vm/reflectioninvocation.cpp

```diff
@@ -2383,6 +2383,137 @@ FCIMPL1(Object*, ReflectionSerialization::GetUninitializedObject, ReflectClassBa
 }
 FCIMPLEND
 
+FCIMPL1(void*, ReflectionSerialization::GetNewobjHelper, ReflectClassBaseObject* objTypeUNSAFE) {
```

Comment:
I think this function should:
- Changed to QCall
- Do all error handling (I do not see why we need `IsFastInstantiable`)
- Return the function pointer and the argument to call the function pointer with (ie no need for the GetMethodTable method) 
- GetUninitializedObject should be switched over to use this